security.sh

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/env bash
 echo ${TEST_URL}
-zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -S -d -u ${SecurityRules} -P 1001 -l FAIL
-curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
-zap-cli --zap-url http://0.0.0.0 -p 1001 report -o /zap/api-report.html -f html
-zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l High --exit-code False
-cp /zap/api-report.html functional-output/
-cat zap.out
\ No newline at end of file
+export PYTHONDONTWRITEBYTECODE=1
+
+echo "Run ZAP scan and generate reports"
+zap-api-scan.py -t ${URL_FOR_SECURITY_SCAN}/v2/api-docs -f openapi -S -d -u ${SECURITY_RULES} -P 1001 -l FAIL --hook=zap_hooks.py -J report.json -r api-report.html
```

Comment:
need to replace ${URL_FOR_SECURITY_SCAN} with ${TEST_URL} variable on line 8 and ${SECURITY_RULES} should be camel case like this ${SecurityRules}

Reply:
Thank you @gokul-sol . Done.